core/server/types/helmchart.go

```diff
@@ -0,0 +1,71 @@
+package types
+
+import (
+	"time"
+
+	"github.com/fluxcd/source-controller/api/v1beta1"
+	pb "github.com/weaveworks/weave-gitops/pkg/api/app"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ProtoToHelmChart(helmChartReq *pb.AddHelmChartReq) v1beta1.HelmChart {
+	labels := map[string]string{
+		ManagedByLabel: managedByWeaveGitops,
+		CreatedByLabel: createdBySourceController,
+	}
+
+	if helmChartReq.AppName != "" {
+		labels[PartOfLabel] = helmChartReq.AppName
+	}
+
+	return v1beta1.HelmChart{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       v1beta1.HelmChartKind,
+			APIVersion: v1beta1.GroupVersion.Identifier(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      helmChartReq.Name,
+			Namespace: helmChartReq.Namespace,
+			Labels:    labels,
+		},
+		Spec: v1beta1.HelmChartSpec{
+			Chart:   helmChartReq.Chart,
+			Version: helmChartReq.Version,
+			SourceRef: v1beta1.LocalHelmChartSourceReference{
+				Kind: helmChartReq.SourceRef.Kind.String(),
+				Name: helmChartReq.SourceRef.Name,
+			},
+			Interval: metav1.Duration{Duration: time.Minute * 1},
```

Comment:
I may be misreading this but it doesn't appear to be using the actual interval

Reply:
This is the default time for interval when we create resources using flux cli. It would be a bit premature to think about using a different value than that. Unless we want to have the default value in the UI and leave the user the option to change it. But as this are prototypes still probably not something we are interested in at this point.